includes/class-amp-story-post-type.php

```diff
@@ -323,6 +329,20 @@ function( $sanitizers ) {
 			}
 		);
 
+		add_filter(
+			'amp_content_sanitizers',
+			function( $sanitizers ) {
+				if ( is_singular( self::POST_TYPE_SLUG ) && isset( $_GET['story_export'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+					$sanitizers['AMP_Story_Export_Sanitizer'] = array(
+						'base_url' => untrailingslashit( AMP_Options_Manager::get_option( 'story_export_base_url' ) ),
+						'slug'     => wp_unslash( $_GET['story_export'] ), // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Is passing `$_GET['story_export']` necessary here? Couldn't this be replaced with `get_post()->post_name`?